WordPress/WordPressShareExtension/Sources/UI/MainShareViewController.swift

```diff
@@ -64,6 +64,7 @@ class MainShareViewController: UIViewController {
 
     override open func viewDidLoad() {
         super.viewDidLoad()
+        BuildSettings.configure(secrets: ApiCredentials.toSecrets())
```

Comment:
I thought of suggesting to do this in the initializer, but it's probably simpler to do it in `viewDidLoad()` after all. Always fiddly to override the init of a view controller.